Wireframe/index.html

```diff
@@ -3,11 +3,47 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <title>WireFrame</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+
   </head>
   <body>
+    <div class="content">
+
+      <header>
+        <h1>Learn Git</h1>
+        <p>Master Git to manage your code, collaborate with teams, and streamline your development process.</p>
+      </header>
+
+      <main class="container">
+
+        <section>
+          <h2>what is git?</h2>
+          <p>Git is a version control system that helps developers track changes in code, collaborate easily, and manage project versions. It allows branching, committing changes, and reverting to earlier versions. Popular platforms like GitHub use Git for smooth teamwork on coding projects.</p>
+          <a href="#" class="read-btn">Read More</a>
+        </section>
+
+        <section class="column">
+          <img src="image/img2.png" alt="git-developer">
+          <h2>why do developers need Git?</h2>
+          <p>Developers need Git to manage code efficiently, especially in teams. It tracks changes, preventing code conflicts when multiple people work on the same files. Git also allows developers to experiment with new features in branches, merge them later, and easily roll back mistakes. It ensures smooth collaboration and keeps a detailed history of project development.</p>
+          <a href="#" class="read-btn">Read More</a>
+        </section>
+
+        <section class="column">
+          <img src="image/img3.jpeg" alt="branch-image">
+          <h2>what is a branch in Git?</h2>
+          <p>A branch in Git is a separate line of development that allows you to work on different features or fixes without affecting the main codebase. You can create a branch to isolate your changes, and once you’re done, merge it back into the main branch (often called “main” or “master”). This helps manage parallel WorkStreams and keeps the code organized and stable.</p>
+          <a href="#" class="read-btn">Read More</a>
+        </section>
+      </main>
+    </div>
     <!-- Add your HTML markup here -->
```

Comment:
Comments that you no longer need, you may remove it. It will make the code more clean.